cli/dcoscli/job/main.py

```diff
@@ -588,8 +591,11 @@ def _add_job(job_file):
         _post_job(full_json)
         job_added = True
     except DCOSHTTPException as e:
-        if e.response.status_code == 409:
-            emitter.publish("Job ID: '{}' already exists".format(job_id))
+        if e.response.status_code == 422:
```

Comment:
this is the main point of this code... the message for http 422 provides a generic processing error in the dcos exception message... however what we want is the information that is in the body of the http response which includes the reason why the json process failed.